docs/versions/releases.json

```diff
@@ -0,0 +1,19 @@
+[
+    {
+        "ReleaseId": "0.0.1",
+        "Artifacts": [
+            {
+                "RID": "linux-x64",
+                "ArtifactURI": "https://pipelines.actions.githubusercontent.com/serviceHosts/04b2e0c6-7a84-40f9-9f91-1f9fee47767d/_apis/pipelines/1/runs/8/signedartifactscontent?artifactName=dnvm%20linux-x64&urlExpires=2022-06-30T18:55:06.6166010Z&urlSigningMethod=HMACV2&urlSignature=QQjhND2v3CcKj4Bg2ZQjYe+1DTRhdnNkjTvg3HxOgIw="
```

Comment:
I think the "urlexpires" stuff in here means we shouldn't be using direct links. GitHub probably wants us to use the release artifact links.